FAQ.md

```diff
@@ -15,3 +30,21 @@ sequelize db:migrate
 ```
 sequelize db:migrate:undo:all
 ```
+
+## Seeders
+
+## How can I create a seeder?
+```
+sequelize seed:create --name seederName 
```

Comment:
Although `seederName` is self sufficient, I will still prefer little bit explanation like https://github.com/sequelize/cli/blob/61104be8e59c0a7ec4cd0517620836d45826e417/FAQ.md#how-can-i-generate-a-model